src/app/containers/MediaPlayer/placeholder/index.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
I've separated this out into its own component as this will be expanded on in the near future (to include a play button with duration).